app/models/manageiq/providers/oracle_cloud/inventory/parser/cloud_manager.rb

```diff
@@ -1,14 +1,75 @@
 class ManageIQ::Providers::OracleCloud::Inventory::Parser::CloudManager < ManageIQ::Providers::OracleCloud::Inventory::Parser
   def parse
-    vms
+    cloud_tenants
+    flavors
+    images
+    instances
   end
 
-  def vms
-    collector.vms.each do |inventory|
-      inventory_object = persister.vms.find_or_build(inventory.id.to_s)
-      inventory_object.name = inventory.name
-      inventory_object.location = inventory.location
-      inventory_object.vendor = inventory.vendor
+  def cloud_tenants
+    collector.compartments.each do |compartment|
+      persister.cloud_tenants.build(
+        :name        => compartment.name,
+        :description => compartment.description,
+        :ems_ref     => compartment.id,
+        :enabled     => compartment.lifecycle_state == "ACTIVE",
+        :parent      => persister.cloud_tenants.lazy_find(compartment.compartment_id)
+      )
+    end
+  end
+
+  def flavors
+    collector.shapes.each do |shape|
+      persister.flavors.build(
+        :ems_ref => shape.shape,
+        :name    => shape.shape,
+        :cpus    => shape.ocpus,
```

Comment:
Is `ocpus` correct?  offhand it looks like a typo

Reply:
`ocpus` is correct (oracle cpus? idk)

![Screenshot from 2021-03-01 11-05-38](https://user-images.githubusercontent.com/12851112/109524129-1e6f2300-7a7e-11eb-90f7-996d31f619e9.png)